src/PageView/WorkoutsMain/WorkoutCurrent/ExcerciseListStarts.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import ExercisesData from '../../../Data/ExercisesData'
+import SmallTimer from '../../Timer/SmallTimer/SmallTimer'
+import './WorkoutCurrent.css'
+
+export default class WorkoutCurrent extends React.Component {
+  constructor () {
```

Comment:
Why change indentation from 4 to 2? Use Prettier extension to format the code (setting: 1 tab = 4 spaces)